package.json

```diff
@@ -133,7 +134,7 @@
     "node-notifier": "^9.0.0",
     "trim-newlines": "^4.0.2",
     "trim": "^0.0.3",
-    "typescript": "^3.9.0",
+    "typescript": "^4.6.2",
```

Comment:
This resolution forces all typescript dependencies to match. Is it still needed?

Reply:
I'm attempting to push forward with parity in order to keep this change as minimally disruptive as possible. My concern is that we end up with unexpected outcomes by removing this in this PR. This and it's existence as a dependency is a solid question and deserving of it's own issue IMO.

Reply:
I'd like to take this in a separate PR, my assumption is that it *is* needed due to the breaking changes within minor versions. We'll need to investigate in full though.

Reply:
OH `api-extractor` uses its own version of Typescript without this. We don't want that.